src/test/resources/feature/SignUp.feature

```diff
@@ -0,0 +1,52 @@
+@Irina
+Feature: SignUp
+
+  Background:
+    Given "Home" is displayed
+    And user clicks on "Sign in"
+    And "Login" is displayed
+    And user clicks on "Email address field"
+    And user enters "lalalala@emailll.com"
+    And user clicks on "Create an account"
+    And "CREATE AN ACCOUNT" is displayed
+
+  Scenario: SignUp - successful
+    When user enters VALID data into mandatory fields:
+      | fields                                        | values                   |
+      | First name                                    | John                     |
+      | Last name                                     | Smitherson               |
+      | Email                                         | j.smithffg@maili.com     |
+      | Password                                      | Ab12%                    |
+      | First name                                    | John                     |
+      | Last name                                     | Smitherson               |
+      | Address                                       | Brunlich, 35001, Broenna |
+      | City                                          | Carrley                  |
+      | State                                         | Alabama                  |
+      | Zip/Postal Code                               | 00000                    |
+      | Country                                       | United States            |
+      | Mobile phone                                  | 105454545                |
+      | Assign an address alias for future reference. | Home address             |
+    And user clicks on "Register"
+    Then "My Account" is displayed
+    And user name is displayed in Navigation bar
+
+  Scenario: SignUp - unsuccessful
+    When user enters INVALID data into mandatory fields:
+      | fields                                        | values      |
+      | First name                                    | Arr457      |
+      | Last name                                     |             |
+      | Email                                         | j.smithffg@ |
+      | Password                                      | A1%         |
+      | First name                                    | Zery5       |
+      | Last name                                     | 7Alkim      |
+      | Address                                       |             |
+      | City                                          |             |
+      | State                                         |             |
+      | Zip/Postal Code                               | A0000       |
+      | Country                                       |             |
+      | Mobile phone                                  |             |
+      | Assign an address alias for future reference. |             |
+    And user clicks on "Register"
+    Then "CREATE AN ACCOUNT" is displayed
+    And errors are displayed
```

Comment:
Here will be nice to specify which errors are displayed

Reply:
Done